qfieldcloud_sdk/cli.py

```diff
@@ -736,3 +736,143 @@ def members_patch(
         log(
             f'Member "{membership["member"]}" changed role in organization "{membership["organization"]}" to role "{membership["role"]}".'
         )
+
+
+@cli.command(name="teams-list", short_help="Get a list of organization teams.")
+@click.argument("organization")
+@click.pass_context
+def teams_list(ctx: Context, organization: str) -> None:
+    """Get a list of organization teams."""
+    teams_list = ctx.obj["client"].get_organization_teams(organization)
+
+    if ctx.obj["format_json"]:
+        print_json(teams_list)
+    else:
+        log(f'Teams members in organization "{organization}":')
+        for team in teams_list:
+            log(f'{team["team"]}')
+
+
+@cli.command(name="teams-create", short_help="Create an organization team.")
+@click.argument("organization")
+@click.argument("team_name")
+@click.pass_context
+def teams_create(ctx: Context, organization: str, team_name: str) -> None:
+    """Create a new team named TEAM_NAME in ORGANIZATION."""
+    team = ctx.obj["client"].create_organization_team(organization, team_name)
+
+    if ctx.obj["format_json"]:
+        print_json(team)
+    else:
+        log(f'Team "{team["team"]}" created in organization "{organization}".')
+
+
+@cli.command(name="teams-get", short_help="Get a list of teams on an organization.")
+@click.argument("organization")
+@click.argument("team_name")
+@click.pass_context
+def teams_get(ctx: Context, organization: str, team_name: str) -> None:
+    """Get details of team TEAM_NAME in ORGANIZATION."""
+    team = ctx.obj["client"].get_organization_team(organization, team_name)
+
+    if ctx.obj["format_json"]:
+        print_json(team)
+    else:
+        log(f'Team "{team_name}" in organization "{organization}":')
+        log(f'  Name: {team["team"]}')
+        log(f'  Organization: {team["organization"]}')
+        log(f'  Members: {", ".join(team["members"])}')
+
+
+@cli.command(name="teams-patch", short_help="Rename an organization team.")
+@click.argument("organization")
+@click.argument("team_name")
+@click.argument("new_team_name")
```

Comment:
```suggestion
@click.option("--name", "new_team_name")
```
This way we can patch other stuff in the future.

Reply:
That's great!... Thanks @suricactus